tools/proteore_build_protein_interaction_maps/build_protein_interaction_maps.xml

```diff
@@ -0,0 +1,311 @@
+<tool id="build_protein_interaction_maps" name="Build Protein interaction network" version="2020.02.06">
```

Comment:
do your scripts, the Python and R ones, have a license? if so you can add those to the `tool` tag.